src/helpers/daily-data-types/apple-health.tsx

```diff
@@ -24,6 +24,17 @@ let appleHealthTypeDefinitions: DailyDataTypeDefinition[] = [
         yAxisConverter: distanceYAxisConverter,
         previewDataRange: [3000, 5000]
     },
+    {
+        type: DailyDataType.AppleHealthEstimatedAppleWatchWearTime,
+        dataProvider: appleHealthEstimatedAppleWatchWearTimeDataProvider,
+        availabilityCheck: simpleAvailabilityCheck("AppleHealth", ["Heart Rate"]),
+        labelKey: "apple-watch-wear-time",
+        icon: <FontAwesomeSvgIcon icon={faHandBackFist} />,
+        formatter: minutesFormatter,
+        yAxisConverter: minutesToHoursYAxisConverter,
+        previewDataRange: [480, 1080],
+        requiresV2Api: true
```

Comment:
I forgot all about this property.  I don't think it actually does anything at the current time and could probably just be dropped.

I think maybe I originally thought we could use something like this to trigger an FYI in VB that extra config was necessary or something when someone selected one of these daily data types.

I think it is fine to leave it here, but I am going to put an item on my TODO list to go back through all the providers and either remove this property all together or update them to all be correct/consistent.  🙂